--- ext/swoole/ext-src/php_swoole.cc.original	2022-11-04 22:52:10
+++ ext/swoole/ext-src/php_swoole.cc	2022-11-04 22:52:44
@@ -302,7 +302,7 @@
 /**
  * use short class name
  */
-STD_ZEND_INI_BOOLEAN("swoole.use_shortname", "On", PHP_INI_SYSTEM, OnUpdateBool, use_shortname, zend_swoole_globals, swoole_globals)
+STD_ZEND_INI_BOOLEAN("swoole.use_shortname", "Off", PHP_INI_SYSTEM, OnUpdateBool, use_shortname, zend_swoole_globals, swoole_globals)
 /**
  * unix socket buffer size
  */
